pyccel/parser/semantic.py

```diff
@@ -2415,6 +2415,48 @@ def _get_for_iterators(self, syntactic_iterable, iterator, new_expr, expr):
 
         return target, iterable
 
+    def env_var_to_pyccel(self, env_var, *, name = None):
+        """
+        Convert an environment variable to a Pyccel AST node.
+
+        Convert an environment variable (i.e. a variable deduced from the
+        context where epyccel was called) into a Pyccel AST node as though
+        the object had been declared explicitly in the code.
+
+        Parameters
+        ----------
+        env_var : object
+            The environemnt variable.
```

Comment:
Typo:
```suggestion
            The environment variable.
```
Why has the spell checker not found it?

Reply:
Hmm, good question. It finds it if I run the command locally :thinking:

Reply:
I'm going to leave the spellchecker for a separate PR because it looks like it is missing other things too :angry: